Source/Public/Restore-Bicep.ps1

```diff
@@ -0,0 +1,32 @@
+function Restore-Bicep {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory, Position = 1)]
+        [ValidateNotNullOrEmpty()]
+        [string]$Path         
+    )
+
+    begin {
+        # Check if a newer version of the module is published
+        if (-not $Script:ModuleVersionChecked) {
+            TestModuleVersion
+        }
+
+        # Verbose output Bicep Version used
+        $FullVersion = Get-BicepNetVersion -Verbose:$false
```

Comment:
Since this output is only shown when running verbose, should we wrap it in a `if ($VerbosePreference)`?

Reply:
Is this actually the command we want to run, as well? `Get-BicepNetVersion` gets the BicepNet version, the wrapper assembly, not the Bicep.Core version.

Reply:
This will output: `VERBOSE: Using Bicep version: 0.4.1008.15138` = The version of Bicep.Core used.